source/_patterns/02-organisms/60-checkout/03-loan-item.mustache

```diff
@@ -1,7 +1,8 @@
-<div class="loan-row row small-collapse large-uncollapse">
+<div class="loan-row row small-collapse">
 	<div class="loan-image small-3 columns">
 		<a class="img thumb" href="{{ loan.url }}">
-			<img src="{{ loan.image.url }}" alt="{{ loan.name }}" title="{{ loan.name }}">
+			<img data-interchange="[{{ loan.image.url }}, (default)], [{{ loan.image.url_2x }}, (large)]" alt="{{ loan.name }}" title="{{ loan.name }}">
```

Comment:
we should probably provide retina sizes as well

Reply:
2x represents retina thumbnail from the qgl query